api/main.js

```diff
@@ -28,6 +50,16 @@ logger.silly('setting up routes');
 routes(server);
 
 logger.silly('starting the server');
-server.listen(process.env.PORT, () =>
-  logger.verbose(`server listening on :${process.env.PORT}`)
-);
+server.listen(process.env.PORT, () => {
```

Comment:
for visibility, i like that these additions are right in-line in `main.js`, but for cleanliness, i wonder if these should be methods in another file (and `server` gets passed in as a param). i'm not sure - just thinking out loud.